funnel/templates/search.html.jinja2

```diff
@@ -24,11 +24,11 @@
               <div class="grid">
                 <div class="grid__col-12">
                   <div class="tabs" id="scrollable-tabs">
-                    <p class="tabs__item tabs__item-control tabs__item-control--prev mui--text-body1 mui--text-light" aria-label="previous" on-click="swipe(-1)"><i class="material-icons tabs__item-control__icon">chevron_left</i></p>
+                    <p class="tabs__item tabs__item-control tabs__item-control--prev mui--text-body1 mui--text-light" aria-label="{% trans %}previous{% endtrans %}" on-click="swipe(-1)"><i class="material-icons tabs__item-control__icon">chevron_left</i></p>
```

Comment:
What is `on-click`? Haven't seen this before. Also, this should have been an `a` if it responds to clicks.

Reply:
Oh, I see it's a ractive attribute.

Reply:
Okay, this is just for scrolling the tabs. No clicking to go anywhere. Nevermind, then.